doc/doc-site/docs/overview/orders.md

```diff
@@ -514,98 +431,91 @@ Market orders and swaps allow users to specify precise trade amounts down to a s
 
 :::
 
-Hence the nominal price of 17792.27 `USD` per `BTC` must be truncated to either 17792.26 or 17792.28.
-For the latter case, this entails an integer price of 889614:
+Next, consider a liquid staking derivative `sAPT` trading against `APT`, `sAPT/APT`, both having 8 decimals.
+Here, very-high price granularity is necessary because the pair is "mean reverting" (to a ratio of 1):
```

Comment:
How about?

"Since the two assets have almost the same nominal price, price granularity will need to be higher here"